app/views/spree/admin/product_feeds/_form.html.erb

```diff
@@ -15,11 +15,8 @@
             { prompt: '' },
             class: "fullwidth select2",
             required: true if @product_catalogs.any? %>
+          <%= link_to t("spree.new_product_catalog"), new_admin_product_catalog_path, class: "btn btn-secondary" %>
```

Comment:
I think this needs to be two-spaces to the left. It's on the same level as `<%= f.label`

Reply:
Nice catch. Thanks!